dashboard/src/app/projects/create-project/create-project.controller.js

```diff
@@ -237,17 +243,23 @@ export class CreateProjectCtrl {
    * Fetching operation has been done, so get workspaces and websocket connection
    */
   updateData() {
-
-    this.workspaces = this.cheAPI.getWorkspace().getWorkspaces();
-
-    // generate project name
-    this.generateProjectName(true);
-
-    // init WS bus
-    if (this.workspaces.length > 0) {
-      this.messageBus = this.cheAPI.getWebsocket().getBus(this.workspaces[0].id);
+    //if create project in progress and workspace have started
+    if (this.createProjectSvc.isCreateProjectInProgress() && this.createProjectSvc.getCurrentProgressStep() > 0) {
+      let workspaceName = this.createProjectSvc.getWorkspaceOfProject();
+      let findWorkspace = this.lodash.find(this.workspaces, function (workspace) {
```

Comment:
arrow function would let use this and avoid to introduce a new local workspaceName variable